src/Relogify.Tests.Unit/TimerTests.fs

```diff
@@ -0,0 +1,81 @@
+module TimerTests
+
+open System
+open Relogify
+open Xunit
+open Timer
+
+[<Fact>]
+let ``Timer is stopped initially`` () =
+    Assert.Equal(NotRunning, Timer.initModel.State)
+
+[<Fact>]
+let ``Extra time is toggled while timer is not running`` () =
+    let model = Timer.initModel
+    let modelWithToggledExtraTime, _, _ = update model ToggleExtraTime
+
+    Assert.Equal(true, modelWithToggledExtraTime.ExtraTime)
+
+let modelReducer (model: Model) (msg: Msg) =
+    let newModel, _, _ = update model msg
+    newModel
+
+[<Fact>]
+let ``Extra time is not toggled while timer is running`` () =
+    let model = Timer.initModel
+    let startTime = DateTime(2000, 01, 01)
+    let actions = [StartRequested; Started startTime; ToggleExtraTime]
+
+    let finalModel = actions |> List.fold modelReducer model
+
+    Assert.Equal(false, finalModel.ExtraTime)
+
+[<Fact>]
+let ``Elapsed time does not get incremented white timer is paused`` () =
+    let model = Timer.initModel
+    let startTime = DateTime(2000, 01, 01, 10, 00, 00)
+    let millisecondsBetweenTicks = 1000.0
+    let millisecondsBetweenTickAndPause = 300.0
```

Comment:
Those are arbitrary numbers, but it might make sense to use FsCheck and write this as a property based test instead:
- Simple version - random `startTime`, `millisecondsBetweenTicks` and `millisecondsBetweenTickAndPause` (with a constraint on `millisecondsBetweenTickAndPause` being smaller than `millisecondsBetweenTicks`).
- More advanced version - random list of actions - starting, ticking, pausing and restarting the timer an arbitrary number of times. The resulting `timeElapsed` should be the sum of ticks, as well as include the times between ticks and pauses.

Reply:
Implemented the simple version